test/test.h

```diff
@@ -9,6 +9,7 @@
 #include <math.h>
 #include <inttypes.h>
 #include <stdarg.h>
+#include <string.h>
```

Comment:
?

Reply:
Just fixed (by using simde_memcmp instead of memcmp directly).